docs/reference/contributing/target/bootstrap.md

```diff
@@ -1,10 +1,64 @@
 ### Bootstrap
 
+#### Bring in CMSIS-Core files
+
 You need CMSIS-CORE files for startup and peripheral memory addresses, and you need linker scripts for Arm, IAR and GCC toolchains. These files are usually in the `mbed-os\targets\TARGET_VENDOR\TARGET_MCU_FAMILY\TARGET_MCUNAME\device` directory.
 
-Mbed OS requires dynamic vector relocation, which requires extensions to CMSIS-CORE. Extend CMSIS-CORE by adding an `mbed-os\targets\TARGET_VENDOR\TARGET_MCUNAME\cmsis.h` file. This header file defines the vector relocation additions and device-specific headers that include CMSIS-CORE. Next add a relocation function in `mbed-os\targets\TARGET_VENDOR\TARGET_MCUNAME\cmsis_nvic.c and .h`. This relocation function changes the contents of the Interrupt Vector Table at run time.
+#### Modify Linker scripts
+
+After adding the core files, the next step is to update the linker scripts for `mbed-os`. To do this, follow the steps below.
+
+* Reserve space for RAM vector table. For the example below this is:
```

Comment:
Query: Do these steps need to happen in order?

Reply:
Nope